sdk/src/test/java/com/silanis/esl/sdk/internal/converter/AccountRoleConverterTest.java

```diff
@@ -0,0 +1,82 @@
+package com.silanis.esl.sdk.internal.converter;
+
+import com.silanis.esl.sdk.AccountRole;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Collections;
+
+public class AccountRoleConverterTest implements ConverterTest {
+
+    @Test
+    public void convertNullSDKToAPI() {
+        AccountRoleConverter accountRoleConverter = new AccountRoleConverter((AccountRole) null);
```

Comment:
Is there any reason to cast null and input it?

Reply:
Yep, as there are 2 constructor with a single parameter, when using null you have to mention which one you want to use.